label_studio/projects/api.py

```diff
@@ -84,18 +85,33 @@
 )
 
 
-class ProjectListAPI(generics.ListCreateAPIView):
-    """
-    get:
-    List your projects
-
+@method_decorator(name='get', decorator=swagger_auto_schema(
```

Comment:
@smoreface let's rewrite docstrings-like helpers to these `@method_decorator`, similarly in other methods like data_import/api, it will allow us to dynamically change URL parameters  like hostname
Also note that by using multiline strings you can freely copy-paste all md content
```
"""
everything is 
multiline here and ```bash work as well
"""
```